src/rgbd_to_ros.cpp

```diff
@@ -52,75 +52,77 @@ int main(int argc, char **argv)
-                {
-                    for(int x = 0; x < view.getWidth(); ++x)
-                    {
-                        geo::Vector3 p;
-                        if (view.getPoint3D(x, y, p))
-                        {
-                            const cv::Vec3b& c = view.getColor(x, y);
-
-                            // Push back and correct for geolib frame
-                            pc_msg->points.push_back(pcl::PointXYZRGB());
-                            pcl::PointXYZRGB& p_pcl = pc_msg->points.back();
-                            p_pcl.x = p.x; p_pcl.y = -p.y; p_pcl.z = -p.z;
-                            p_pcl.r = c[2]; p_pcl.g = c[1]; p_pcl.b = c[0];
-                            pc_msg->width++;
-//                            std::cout << p << std::endl;
-                        }
-                        else
-                        {
-//                            pc_msg->points.push_back(pcl::PointXYZ(0, 0, 0));
-//                            pc_msg->width++;
-                            pc_msg->is_dense = false;
-                        }
-                    }
-                }
-
-                // Publish
-                pub_depth_pc.publish(pc_msg);
+//                 // Create point cloud
+//                 pcl::PointCloud<pcl::PointXYZRGB>::Ptr pc_msg(new pcl::PointCloud<pcl::PointXYZRGB>());
+
+//                 pc_msg->header.stamp = image.getTimestamp() * 1e6;
+//                 pc_msg->header.frame_id = image.getFrameId();
+//                 pc_msg->width  = 0;
+//                 pc_msg->height  = 1;
+//                 pc_msg->is_dense = true;
+
+//                 // Fill point cloud
+//                 for(int y = 0; y < view.getHeight(); ++y)
+//                 {
+//                     for(int x = 0; x < view.getWidth(); ++x)
+//                     {
+//                         geo::Vector3 p;
+//                         if (view.getPoint3D(x, y, p))
+//                         {
+//                             const cv::Vec3b& c = view.getColor(x, y);
+
+//                             // Push back and correct for geolib frame
+//                             pc_msg->points.push_back(pcl::PointXYZRGB());
+//                             pcl::PointXYZRGB& p_pcl = pc_msg->points.back();
+//                             p_pcl.x = p.x; p_pcl.y = -p.y; p_pcl.z = -p.z;
+//                             p_pcl.r = c[2]; p_pcl.g = c[1]; p_pcl.b = c[0];
+//                             pc_msg->width++;
+// //                            std::cout << p << std::endl;
+//                         }
+//                         else
+//                         {
+// //                            pc_msg->points.push_back(pcl::PointXYZ(0, 0, 0));
+// //                            pc_msg->width++;
+//                             pc_msg->is_dense = false;
+//                         }
+//                     }
+//                 }
+
+//                 // Publish
+//                 pub_depth_pc.publish(pc_msg);
```

Comment:
Maybe we can create a separate node for this, because it is unwanted to run this continuously.